Sources/App/Utilities/SemverVersion.swift

```diff
@@ -86,24 +107,29 @@ struct SemverVersion {
     func next(_ bump: Bump) -> SemverVersion {
         var version = self
         if var prerelease = version.prerelease {
+            let p: Prerelease?
             switch bump {
+            case .stage:
+                p = prerelease.bump
```

Comment:
Couldn't this be `prerelease = prerelease.nextRelease`?

Reply:
`.nextRelease` is optional, so it can't. It's why I added the local var, but I can skip that and assign directly to `version.prerelease`